### PR TITLE
Version bump to v2.0.0

### PR DESCRIPTION
**Pull request by avestuk**

### CODE REVIEW

File: k8s/deploy-storageos/upgrade-helper/prepare-upgrade.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-LATEST_VERSION="storageos/node:1.5.3"
+LATEST_VERSION="storageos/node:v2.0.0"
```
Comment (Arau) [inline k8s/deploy-storageos/upgrade-helper/prepare-upgrade.sh]: I don't think we should change this

File: systemd-service/storageos.service [hunk 2]
```diff
@@ -11,8 +11,8 @@ TimeoutStartSec=0
 EnvironmentFile=/etc/default/storageos
 ExecStartPre=-/usr/bin/docker kill storageos
 ExecStartPre=-/usr/bin/docker rm storageos
-ExecStartPre=/usr/bin/docker pull storageos/node:1.5.3
-ExecStart=/usr/bin/docker run --name storageos -e DISABLE_TELEMETRY=true -e HOSTNAME -e ADVERTISE_IP -e JOIN --net=host --pid=host --privileged --cap-add SYS_ADMIN --device /dev/fuse -v /var/lib/storageos:/var/lib/storageos:rshared -v /run/docker/plugins:/run/docker/plugins -v /sys:/sys storageos/node:1.5.3 server
+ExecStartPre=/usr/bin/docker pull storageos/node:v2.0.0
```
Comment (Arau) [inline systemd-service/storageos.service]: We should not change this. We don't  support this type of installation for v2

**Review by Arau — COMMENTED** [review]
Do we want to change it everywhere?

File: k8s/deploy-storageos/upgrade-helper/prepare-upgrade.sh (hunk 1 above)
Comment (avestuk) [reply k8s/deploy-storageos/upgrade-helper/prepare-upgrade.sh]: Agreed probably a bad idea.

File: systemd-service/storageos.service (hunk 2 above)
Comment (avestuk) [reply systemd-service/storageos.service]: Yeap fair point.

**Review by Arau — APPROVED** [review]

**Pull request merged**